ajax/libs/twbuttons/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "twbuttons",
+  "filename": "twbuttons.css",
+  "description": "alexwolfe/Buttons for Twitter Bootstrap 3",
+  "license": "MIT",
+  "keywords": [
+    "buttons",
+    "less",
+    "css",
+    "css buttons",
+    "less buttons",
+    "bootstrap"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/onokumus/twbuttons.git"
+  },
+  "homepage": "https://onokumus.github.io/twbuttons/",
+  "author": "Osman Nuri Okumuş <onokumus@gmail.com> (https://github.com/onokumus)",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/onokumus/twbuttons.git",
+    "basePath": "dist"
```

Comment:
Please change `auto-update` config so that our bot could correctly fetch the css files
```
"autoupdate": {
    "source": "git",
    "target": "git://github.com/onokumus/twbuttons.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/*"
        ]
      }
    ]
  }
```